README.md

```diff
@@ -1,177 +1,230 @@
+[🤖 Assistant](#ui-assistant) &nbsp;&nbsp;|&nbsp;&nbsp; 
+[🚀 Launch Experiments](#🚀-launch-experiments) &nbsp;&nbsp;|&nbsp;&nbsp;
+[🔍 AgentXray](#🔍-agentxray) &nbsp;&nbsp;|&nbsp;&nbsp; 
+[🤖 Make Your Own Agent](#implement-a-new-agent) &nbsp;&nbsp;|&nbsp;&nbsp;
+[↻ Reproducibility](#↻-reproducibility) &nbsp;&nbsp;|&nbsp;&nbsp;
+
+<video controls style="max-width: 800px;">
+  <source src="https://github.com/ServiceNow/BrowserGym/assets/26232819/e0bfc788-cc8e-44f1-b8c3-0d1114108b85" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
 
 AgentLab is a framework for developing and evaluating agents on a variety of
-benchmarks supported by [BrowserGym](https://github.com/ServiceNow/BrowserGym).
-This includes:
-* WebArena
-* WorkArena.L1, L2, L3
-* VisualWebArena (coming soon...)
-* MiniWoB
-
-The framework enables the desing of rich hyperparameter spaces and the launch of
-parallel experiments using ablation studies or random searches. It also provides
-agent_xray, a visualization tool to inspect the results of the experiments using
-a custom gradio interface
-
-<a href="https://github.com/user-attachments/assets/20a91e7b-94ef-423d-9091-743eebb4733d">
-  <img src="https://github.com/user-attachments/assets/20a91e7b-94ef-423d-9091-743eebb4733d" width="250" />
-</a>
-
-## Install agentlab
-
-This repo is intended for testing and developing new agents, hence we clone and install using the `-e` flag.
+[benchmarks](#🎯-supported-benchmarks) supported by
+[BrowserGym](https://github.com/ServiceNow/BrowserGym).
+
+AgentLab Features:
+* Easy large scale parallel agent experiments using [ray](https://www.ray.io/)
+* Building blocks for making agents
+* Unified LLM api for OpenRouter, OpenAI, Azure, or self hosted using TGI.
+* Prefered way for running benchmarks like WebArena
+* Various Reproducibility features
+* Unified LeaderBoard (soon)
+
+## 🎯 Supported Benchmarks
+| Benchmark | Setup  <br> Link | # Task <br> Template| Seed  <br> Diversity | Max  <br> Step | Multi-tab | Hosted Method | BrowserGym <br> Leaderboard |
+|-----------|------------|---------|----------------|-----------|-----------|---------------|----------------------|
+| [WebArena](https://webarena.dev/) | [setup](https://github.com/ServiceNow/BrowserGym/blob/main/browsergym/webarena/README.md) | 812 | None | 30 | yes | self hosted (docker) | soon |
+| [WorkArena](https://github.com/ServiceNow/WorkArena) L1 | [setup](https://github.com/ServiceNow/WorkArena?tab=readme-ov-file#getting-started) | 33 | High | 30 | no | demo instance | soon |
+| [WorkArena](https://github.com/ServiceNow/WorkArena) L2 | [setup](https://github.com/ServiceNow/WorkArena?tab=readme-ov-file#getting-started) | 341 | High | 50 | no | demo instance | soon |
+| [WorkArena](https://github.com/ServiceNow/WorkArena) L3 | [setup](https://github.com/ServiceNow/WorkArena?tab=readme-ov-file#getting-started) | 341 | High | 50 | no | demo instance | soon |
```

Comment:
```suggestion
| [WorkArena L1](https://github.com/ServiceNow/WorkArena) | [setup](https://github.com/ServiceNow/WorkArena?tab=readme-ov-file#getting-started) | 33 | High | 30 | no | demo instance | soon |
| [WorkArena L2](https://github.com/ServiceNow/WorkArena) | [setup](https://github.com/ServiceNow/WorkArena?tab=readme-ov-file#getting-started) | 341 | High | 50 | no | demo instance | soon |
| [WorkArena L3](https://github.com/ServiceNow/WorkArena) | [setup](https://github.com/ServiceNow/WorkArena?tab=readme-ov-file#getting-started) | 341 | High | 50 | no | demo instance | soon |
```
i just feel like it looks weird if only part of the name is an hyperlink (https://github.com/ServiceNow/AgentLab/tree/refine-ux)